src/Farmer/Builders/Builders.Sql.fs

```diff
@@ -7,124 +7,127 @@ open Farmer.Sql
-    member this.ConnectionString =
+    member this.ConnectionString (database:SqlAzureDbConfig) =
         concat
             [ literal
                 (sprintf "Server=tcp:%s.database.windows.net,1433;Initial Catalog=%s;Persist Security Info=False;User ID=%s;Password="
-                    this.ServerName.ResourceName.Value
                     this.Name.Value
+                    database.Name.Value
                     this.AdministratorCredentials.UserName)
               this.AdministratorCredentials.Password.AsArmRef
               literal ";MultipleActiveResultSets=False;Encrypt=True;TrustServerCertificate=False;Connection Timeout=30;" ]
-    member this.Server =
-        this.ServerName.ResourceName
+    member this.ConnectionString databaseName =
+        this.Databases
+        |> List.tryFind(fun db -> db.Name = databaseName)
+        |> Option.map this.ConnectionString
+        |> Option.defaultWith(fun _ -> failwithf "Unknown database name %s" databaseName.Value)
+    member this.ConnectionString databaseName = this.ConnectionString (ResourceName databaseName)
+
     interface IBuilder with
-        member this.DependencyName = this.Server
+        member this.DependencyName = this.Name
         member this.BuildResources location resources = [
-            let elasticPoolName = this.Server.Map (sprintf "%s-pool")
-            let database =
-                {| Name = this.Name
-                   Sku =
-                    match this.Sku with
-                    | PoolSku _
-                    | ExternalPool _ ->
-                        Pool elasticPoolName
-                    | DbSku sku ->
-                        Standalone sku
-                   Collation = this.DbCollation
-                   TransparentDataEncryption = this.Encryption |}
-
-            match this.ServerName with
-            | AutomaticallyCreated serverName ->
-                { ServerName = serverName
-                  Location = location
-                  Credentials =
-                    {| Username = this.AdministratorCredentials.UserName
-                       Password = this.AdministratorCredentials.Password |}
-                  FirewallRules = this.FirewallRules
-                  ElasticPool =
-                    match this.Sku with
-                    | PoolSku sku ->
-                        Some {| Name = elasticPoolName; Sku = sku |}
-                    | DbSku _
-                    | ExternalPool ->
-                        None
-                  Databases = [ database ]
-                }
-            | External serverName ->
-                resources
-                |> Helpers.mergeResource serverName (fun server -> { server with Databases = database :: server.Databases })
-            | AutomaticPlaceholder ->
-                failwith "SQL Server Name has not been set."
+            let elasticPoolName = this.Name.Map (sprintf "%s-pool")
```

Comment:
For my use case I need to be able to override this and give it a name of my choosing.

Reply:
Done.